ovirt_imageio/_internal/server.py

```diff
@@ -177,3 +184,17 @@ def _drop_privileges(self):
         os.initgroups(self.config.daemon.user_name, gid)
         os.setgid(gid)
         os.setuid(uid)
+
+    def _add_ticket(self, ticket):
+        log.info("Initial ticket: %s", ticket)
+        self.auth.add(self._read_ticket(ticket))
+
+    def _read_ticket(self, ticket):
+        try:
+            with open(ticket, 'r') as f:
+                return json.loads(f.read())
+        except ValueError as e:
+            raise errors.InvalidTicket(
+                f"Ticket is not in a json format: {e}") from e
+        except FileNotFoundError as e:
+            raise errors.InvalidTicket(f"Ticket not found: {e}") from e
```

Comment:
The issue is missing file, so maybe:

    f"Cannot read ticket {ticket}: {e}"

(The ticket path may not be needed if it is included in e, I did not check)